src/main/resources/db/migration/V11__Add_journalposttype_to_journalfoeringdata.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE document.journalfoeringdata
+    ADD COLUMN IF NOT EXISTS journalposttype TEXT;
```

Comment:
"if not exists" skulle ikke være nødvendig på "nye" migreringsskript? Eller har denne også kjørt flere ganger?

Reply:
Kjørte det først i dev uten slettingen av dokumentenhet.should_be_distributed. Foreslår å gjøre det slik nå, men er veldig med på at jeg legger om prosessen litt når det gjelder dev.